mcmd/commands/delete.py

```diff
@@ -0,0 +1,81 @@
+import mcmd.config.config as config
+from mcmd import io
+from mcmd.client.molgenis_client import login, delete, delete_data, resource_exists, ResourceType
+from mcmd.io import highlight
+from mcmd.utils import McmdError
+
+
+# =========
+# Arguments
+# =========
+
+def arguments(subparsers):
+    p_delete = subparsers.add_parser('delete',
+                                     help='Delete entities or data',
+                                     description="Run 'mcmd delete entity -h' or 'mcmd delete data -h' to view the help"
+                                                 " for those sub-commands")
+    p_delete_subparsers = p_delete.add_subparsers(dest="type")
+    p_delete_entity = p_delete_subparsers.add_parser('entity',
```

Comment:
Let's change this to `entity-type` to make sure nobody trips over the definitions. Sorry for changing my mind again 😬

Reply:
Entity-type is long and inconvenient when using python, enable and disable also use entity as argument already